contrib/babelfishpg_common/src/collation.h

```diff
@@ -115,34 +115,40 @@ typedef struct collation_callbacks
 
 	const char *(*translate_bbf_collation_to_tsql_collation) (const char *collname);
 
+	const char *(*translate_tsql_collation_to_bbf_collation) (const char *collname);
+
+	void		(*set_db_collation) (Oid db_coll);
+
 } collation_callbacks;
 
 extern int	find_cs_as_collation(int collidx);
 extern int	find_any_collation(const char *collation_name, bool check_for_server_collation_name_guc);
-extern Oid	get_server_collation_oid_internal(bool missingOk);
+extern Oid	get_database_or_server_collation_oid_internal(bool missingOk);
 extern coll_info lookup_collation_table(Oid collid);
 extern int8_t cmp_collation(uint16_t coll1, uint16_t coll2);
-extern bool collation_is_CI_AS(Oid colloid);
```

Comment:
For this part, why do we need to change collation_is_CI_AS to collation_is_CI ?
Why not just adding a new function ?

Reply:
Update this directly is not that straight forward, few sql function depends of this C function hence we need to be careful for upgrade scenario. Hence, we did not fix that in this PR, we have opened an issue to track that task.